quartz/src/main/java/org/quartz/xml/ValidationException.java

```diff
@@ -127,10 +126,6 @@ public String getMessage() {
         boolean first = true;
 
         for (Exception e : getValidationExceptions()) {
-            if (!first) {
-                sb.append('\n');
-                first = false;
```

Comment:
This looks to be a bug where this line belongs in an else.

Reply:
Agreed, it does look like a bug